app/js/reducers/initialState.js

```diff
@@ -108,4 +108,8 @@ export default {
     dateFormat: '',
     error: null,
   },
+  labOrderables: {
+    error: false,
+    loading: false,
+  },
```

Comment:
You only have properties in the state(`labOrderables` to store `error` and track `loading`, where are the `labOrderables` themselves to be stored after fetching them?

Reply:
Done, Thanks.